### DISCUSSION TITLE
no countdown for restoring NVRAM

### DISCUSSION THREAD

**Discussion by lionel846** [exit-failure/tds7000#3] [Q&A]
Hi
Recently I got a TDS7054 which does not boot. I plugged the serial port and as expected it displayed the error on NVRAM.
I checked the battery, it's empty. I changed it and then tried to restore parameters as you said, with the console. But I have no countdown, and cannot see any menu.
What did I miss?
Thank you for your help and advice.

**Comment by exit-failure** [comment]
Sorry for replaying so late :-/
Maybe there's something else wrong with your scope. Does it give you any serial output at all?
